shared/src/main/scala/coop/rchain/shared/Language.scala

```diff
@@ -11,4 +11,10 @@ object Language {
     val _: A = a
     ()
   }
+
+  def removeIndex[E](col: Seq[E], index: Int): Seq[E] = {
+    val (l1, l2) = col splitAt index
+    (l1 ++ l2.drop(1))
+  }
```

Comment:
We have this function duplicated on three places. Also working with index on a Seq is not advisable because it can be slow (linear with elements).
For now at least we can use this one here until we investigate better solution.